packages/remme-public-key-storage/src/index.ts

```diff
@@ -103,6 +113,19 @@ class RemmePublicKeyStorage implements IRemmePublicKeyStorage {
         }
     }
 
+    private async _constructKeysFromPayload(
+        payload: NewPubKeyPayload | INewPubKeyPayload,
+    ): Promise<IRemmeAccount> {
+        const { rsa, ecdsa, ed25519 } = payload;
+        const { key } = rsa || ecdsa || ed25519;
+        return await RemmeKeys.construct({
+            keyType: rsa ? KeyType.RSA :
+                     ecdsa ? KeyType.ECDSA :
+                     ed25519 ? KeyType.EdDSA : undefined,
+            publicKey: key,
+        });
```

Comment:
change to use "configuration" property of an object.

F.I.
const encode = NewPubKeyPayload.create({
    entityHash: "sad",
    entityHashSignature: "sfs",
    ecdsa: new NewPubKeyPayload.ECDSAConfiguration({
      key: "sdaf"
    }),
  });
  console.log(encode[encode.configuration].key);